lib/ecto/migration.ex

```diff
@@ -420,7 +420,7 @@ defmodule Ecto.Migration do
             name: String.t(),
             prefix: atom | nil,
             comment: String.t() | nil,
-            primary_key: boolean | keyword(),
+            primary_key: boolean | keyword() |  [start_value: integer(), increment: integer(), cache: pos_integer(), clause: :by_default | :always],
```

Comment:
I think this should remain as keyword, as we don't document the other options here anyway (and this would need to include type and so on). But we need to find a place to document all identity options. Perhaps something in the module docs about the identity type.

Reply:
I understand. Are you open to eventually down the line add specs? I started on it but had to stop. 

I can never remember options.